ntbs-service/Pages/ClinicalTimelines/Edit.cshtml.cs

```diff
@@ -67,13 +80,12 @@ public async Task<IActionResult> OnPostAsync(int? id)
 
             if (!ModelState.IsValid)
             {
-                return await OnGetAsync(id);
+                await OnGetAsync(NotificationId);
             }
 
-            var notification = await service.GetNotificationAsync(id);
+            var notification = await service.GetNotificationAsync(NotificationId);
             await service.UpdateTimelineAsync(notification, ClinicalTimeline);
-
-            return RedirectToPage("/Patients/Index");
+            return true;

```

Comment:
In general I've seen you shouldn't return void from an async function but this function doesn't return anything interesting - what's the correct way to do this?

Reply:
Yes I think just don't have a return line should be fine?